src/actions/Action.container.tsx

```diff
@@ -53,6 +57,12 @@ function ActionContainer({ name }: ActionContainerProps) {
     message.success(`Action '${name}' run successfully`);
   };
 
+  const deleteAction = () => {
+    deleteActionMutation.mutate({ actionName: name });
+    message.success(`Action '${name}' deleted successfully`);
```

Comment:
```suggestion
    message.success(`Action '${name}' deletion successfully scheduled`);
```

or
```suggestion
    message.success(`Successfully scheduled Action '${name}' deletion`);
```


as you don't wait until successful deletion and this delete is also not a force delete, so sometimes it can be ignored.